tox.ini

```diff
@@ -165,6 +165,8 @@ filterwarnings =
     ignore::pytest.PytestExperimentalApiWarning
     # Do not cause SyntaxError for invalid escape sequences in py37.
     default:invalid escape sequence:DeprecationWarning
+    # ignore use of unregistered marks, because we use many to test the implementation
+    ignore::_pytest.warning_types.UnknownMarkWarning
```

Comment:
I don't think this is necessary anymore given that you changed all `pytest.mark.issueX` marks, correct?

Reply:
Unfortunately there are also many, many other unregistered marks used to test the marks machinery, including (but not limited to) `foo`, `bar`, `baz`, `xyz`, `XYZ`, `nothing`, various spellings of `parametrize`, etc.

After trying to register them - see the earlier commits and CI logs - I eventually decided that it would be easier to specialize the warning type and ignore it.